<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardDialog</class>
 <widget class="QDialog" name="BoardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>제목</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleLineEdit">
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="questionLabel">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>질문</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>305</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="questionPlainTextEdit">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35100</width>
         <height>510</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>351</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;나눔바른고딕&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="answerLabel">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>답변</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>308</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="answerPlainTextEdit">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35100</width>
         <height>221000</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>351</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;나눔바른고딕&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>2</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>70</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>확  인</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>3</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>취  소</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>70</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
